backend/src/main/scala/cromwell/backend/io/GlobFunctions.scala

```diff
@@ -0,0 +1,35 @@
+package cromwell.backend.io
+
+import java.nio.file.Files
+
+import cromwell.backend.BackendJobDescriptor
+import cromwell.core.CallContext
+import wdl4s.TaskCall
+import wdl4s.expression.{NoFunctions, PureStandardLibraryFunctionsLike}
+import wdl4s.values._
+import scala.collection.JavaConverters._
+import cromwell.core.path.PathImplicits._
+
+trait GlobFunctions extends PureStandardLibraryFunctionsLike {
+
+  def callContext: CallContext
+
+  def findGlobOutputs(call: TaskCall, jobDescriptor: BackendJobDescriptor): Set[WdlGlobFile] = {
+    val globOutputs = call.task.findOutputFiles(jobDescriptor.fullyQualifiedInputs, NoFunctions) collect {
+      case glob: WdlGlobFile => glob
```

Comment:
I like collect better than an explicit filter, then map from `WdlFile` to `WdlGlobFile`.